pages/library/_viewId.vue

```diff
@@ -1,87 +1,61 @@
 <template>
-  <v-container>
-    <v-row class="align-center">
-      <v-col cols="9">
-        <v-select
-          v-model="orderMethod"
-          class="ma-2"
-          :items="sortChoices"
-          autowidth
-          @change="sortItems"
-        ></v-select>
-      </v-col>
-      <v-col>
-        <v-btn class="ma-2" @click="changeSortDirection"
-          ><v-icon :class="{ flipped: sortDirection }"
-            >mdi-arrow-up</v-icon
-          ></v-btn
-        >
-      </v-col>
-
-      <v-col>
-        <filter-button
-          :collection-info-item="collectionInfoItem"
-          @change="filterMedia"
-        />
-      </v-col>
-    </v-row>
-    <v-row v-if="!loaded">
-      <v-col cols="12" class="card-grid-container">
-        <skeleton-card v-for="n in 24" :key="n" />
-      </v-col>
-    </v-row>
-    <dynamic-scroller
-      v-if="items.length"
-      class="scroller"
-      :items="itemsChunks"
-      :min-item-size="350"
-      :buffer="$vuetify.breakpoint.height * 1.5"
-      page-mode
-    >
-      <template #default="{ item, index, active }">
-        <dynamic-scroller-item
-          :item="item"
-          :active="active"
-          :data-index="index"
-          class="card-grid-container"
-        >
-          <card v-for="card of item.chunk" :key="card.Id" :item="card" />
-        </dynamic-scroller-item>
-      </template>
-    </dynamic-scroller>
-    <v-row v-else-if="loaded" justify="center">
-      <v-col cols="12" class="card-grid-container empty-card-container">
-        <skeleton-card v-for="n in 24" :key="n" boilerplate />
-      </v-col>
-      <div class="empty-message text-center">
+  <div>
+    <v-app-bar fixed flat dense class="second-toolbar">
+      <span class="text-h6 hidden-sm-and-down">
+        {{ collectionInfo.Name }}
+      </span>
+      <v-chip small class="ma-2 hidden-sm-and-down">{{ itemsCount }}</v-chip>
+      <v-divider inset vertical class="mx-2 hidden-sm-and-down" />
+      <type-button
+        v-if="collectionInfo.CollectionType"
+        :type="collectionInfo.CollectionType"
+        @change="onChangeType"
+      />
+      <v-divider v-if="isSortable" inset vertical class="mx-2" />
+      <sort-button v-if="isSortable" @change="onChangeSort" />
+      <filter-button
+        v-if="isSortable"
+        :collection-info="collectionInfo"
+        :items-type="viewType"
+        @change="onChangeFilter"
+      />
+    </v-app-bar>
+    <v-container class="after-second-toolbar">
+      <item-grid v-if="items.length" :loading="loading" :items="items">
         <h1 class="text-h5">
           {{ $t('libraryEmpty') }}
         </h1>
-      </div>
-    </v-row>
-  </v-container>
+      </item-grid>
```

Comment:
```suggestion
      <item-grid v-if="items.length" :loading="loading" :items="items" />
```
It shows an empty library error in the itemGrid?

Reply:
The condition is already inside of ItemGrid, so I removed it from here.